_includes/pbc/all/upgrade-modules/upgrade-the-productconfigurationstorage-module.md

```diff
@@ -1,4 +1,26 @@
+## Upgrading from version 0.2.* to version 1.0.*
 
+*Estimated migration time: 5 minutes*
+
+`ProductConfigurationStorage` v1.0.0 introduces the following backward incompatible changes:
+
+* Replaced ProductConfigurationStorageClientInterface::findProductConfigurationInstanceBySku() with ProductConfigurationStorageClientInterface::getProductConfigurationInstanceCollection().
+* Replaced ProductConfigurationStorageClientInterface::findProductConfigurationInstancesIndexedBySku() with ProductConfigurationStorageClientInterface::getProductConfigurationInstanceCollection().
```

Comment:
Please say which methods we removed and which added. This is the actual module adding the methods.